.github/workflows/deploy.yml

```diff
@@ -0,0 +1,25 @@
+name: Build and Publish Image
+
+on:
+  push:
+    branches:
+      - "main"
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  build-publish-image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: "Build and Publish Image"
+        run: |
+          docker build . --tag ghcr.io/kids-first/kf-api-dataservice/kf-api-dataservice:post-release-${ git tag --sort=taggerdate | tail -1 | head -1 }-${GITHUB_SHA:0:7}
```

Comment:
So this gives us the latest-release tag which we also apply to the latest release. Also for some reason git tag --sort=taggerdate doesn't sort by date
```
git tag --sort=taggerdate | tail -1 | head -1 
```

This seems to work: 
```
git tag --sort=committerdate | grep -E '[0-9]' | tail -1 | cut -b 1-7
```
It'll give us 1.32.0 right now. Modified it from https://gist.github.com/rponte/fdc0724dd984088606b0?permalink_comment_id=3180306#gistcomment-3180306

Reply:
will update